dexterity/membrane/configure.zcml

```diff
@@ -27,6 +27,14 @@
        are visible and can be installed separately.  Any upgrade steps
        for that profile are also defined there. -->
 
+  <genericsetup:registerProfile
+      name="uninstall"
+      title="dexterity.membrane: behaviors"
```

Comment:
Can you add 'uninstall' somewhere in the title?

Reply:
done ;-)